apps/application-status/pages/index.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <div>
+    <navbar />
+    <div class="metrics flex flex-col self-center">
+      <div>
+        <b class="text-3xl">{{ entities.sprint.name }}</b>
+      </div>
+      <br>
+      <div>
+        <b class="text-2xl">ENTITIES REPO</b>
+      </div>
+      <div>
+        <h2 class="text-xl">
+          Total Points
+        </h2>
+        <pre>{{ entities.sprint.totalPoints }}</pre>
+      </div>
+      <div>
+        <h2 class="text-xl">
+          Total closed releases
+        </h2>
+        <pre>{{ entities.sprint.numberOfRelease }}</pre>
+      </div>
+      <div>
+        <h2 class="text-xl">
+          Total Issues
+        </h2>
+        <pre>{{ calculateTotalIssue(entities) }}</pre>
+      </div>
+      <br>
+      <div>
+        <b class="text-2xl">NAME TEAM REPO</b>
```

Comment:
A bit confused about this file, how do I view it from the main page using the temp gcbrun deployment? Are NAME TEAM REPO and ENTITIES REPO just placeholders?

Reply:
This is the current display of the page, 
![image](https://github.com/bcgov/sbc-producthub/assets/144156265/3008a679-0422-4bb8-bddd-184d76e94f3c)

It's a MVP product and current stage is just to display the correct information on screen before taking UI into consideration so I haven't styled it yet